modules/topology_hiding/topo_hiding_logic.c

```diff
@@ -2100,7 +2114,15 @@ static int topo_no_dlg_seq_handling(struct sip_msg *msg,str *info)
 		free_rr(&head);
 	pkg_free(dec_buf);
 
-	if (topo_no_dlg_encode_contact(msg,0,NULL) < 0) {
+	if (flags_buf.len && flags_buf.s) {
+		if (sscanf(flags_buf.s, "%d", &flags) < 0) {
```

Comment:
similarly, the practice is to use the [int2str](https://github.com/OpenSIPS/opensips/blob/master/ut.h#L720) function for this.